FND/Accounting/AccountingEquity.rdf

```diff
@@ -124,7 +122,8 @@
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label xml:lang="en">income</rdfs:label>
-		<skos:definition>cash or cash equivalent(s) received during a period of time in exchange for labor or services, from the sale of goods or property, or as profit from financial investments</skos:definition>
+		<skos:definition>revenue received during a period of time</skos:definition>
+		<fibo-fnd-utl-av:explanatoryNote>Income includes cash or cash equivalent(s) received during some period of time in exchange for labor or services, from the sale of goods or property, or as profit from financial investments.</fibo-fnd-utl-av:explanatoryNote>
```

Comment:
Income from a financial investment may not constitute profit, depending on how much was invested relative to the return.

Reply:
Changed to receipts.